src/partials/error.hbs

```diff
@@ -0,0 +1,25 @@
+<div class="icon-feedback icon-feedback--error">
+  <span class="icon-feedback__icon pingicon-ghost-error icon-feedback--error"></span>
+  <div class="icon-feedback__label">
+    {{#if errorHeader}}
+      <p class="icon-feedback--error">{{errorHeader}}</p>
+    {{else}}
+      <p class="icon-feedback--error">Error</p>
```

Comment:
This is showing up for MFA_FAILED states, but I believe UX would prefer if "Blocked" was the error header for those dead-end failures (and "Error" for scenarios that are recoverable)

Reply:
done, `mfa_failed` state now passes in an errorHeader